hr_attendance_management/models/hr_employee.py

```diff
@@ -232,9 +232,10 @@ def _cron_compute_annual_balance(self):
             employee.penultimate_period_lost_hours = \
                 employee.previous_period_lost_hours
             employee.previous_period_continuous_cap = \
-                employee.extra_hours_extra_hours_continuous_cap
+                employee.extra_hours_continuous_cap
+
+            new_balance = employee.balancejesusislove
```

Comment:
Is it an easter egg introducing a bug? Or is it solving all our problems? :smile: